nautobot/extras/tests/test_customfields.py

```diff
@@ -116,36 +111,34 @@ def test_simple_fields(self):
         obj_type = ContentType.objects.get_for_model(Location)
 
         for data in DATA:
-
             # Create a custom field
             # 2.0 TODO: #824 slug rather than name
-            cf = CustomField(type=data["field_type"], name="my_field", required=False)
+            cf = CustomField(type=data["field_type"], label="my_field", required=False)
             cf.save()  # not validated_save this time, as we're testing backwards-compatibility
             cf.content_types.set([obj_type])
             # Assert that slug and label were auto-populated correctly
             # 2.0 TODO: slug and label will become mandatory fields to specify.
             cf.refresh_from_db()
-            self.assertEqual(cf.label, cf.name)
-            self.assertEqual(cf.slug, cf.name)
+            self.assertEqual(cf.slug, cf.label)
```

Comment:
Probably should change this test to use a human-readable `label` and assert that `cf.slug` equals `slugify_dashes_to_underscores(cf.label)`?